src/components/templates/Jazzbar/JazzTab/JazzTab.scss

```diff
@@ -20,7 +20,7 @@
   .music-bar-content {
     display: grid;
 
-    padding: 10px 70px;
+    padding: 10px 70px 40px;
```

Comment:
fixed bottom indent for tables on the `Jazzbar` template. They were cropped.

<img width="1431" alt="Screenshot 2021-07-16 at 13 26 55" src="https://user-images.githubusercontent.com/26366773/125934356-c5114dd4-0d15-4a9d-9dff-1c9acbaf93c2.png">